app/locales/zh-tw/translations.coffee

```diff
@@ -65,6 +65,39 @@ I18nTranslationsZhTw =
         "start_receiving_by": "{{lastName}} {{firstName}} 正在接收"
         "inactive": "列作暫停"
 
+    offer_filters:
+      time_presets:
+        overdue: "Overdue",
+        today: "Today",
+        tomorrow: "Tomorrow",
+        week: "This week",
+        next_week: "Next week",
+        month: "This month",
+        next_month: "Next month"
+
+      self_review: "Mine"
+      all_review: "All"
+      time_filter_title: "Offer Due Date"
+      button_state: "State"
+      showPriority: "Show Priority"
```

Comment:
I think some translations are already in Stock, maybe you can copy them over 

I notice this in stock
```
    showPriority: "只顯示優先處理訂單",
    showPriority_info: "優先訂單已停留在一狀態過長，需要盡快處理",
```